download-box.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
 if [ ! -f box.phar ]; then
-    wget https://github.com/humbug/box/releases/download/3.1.0/box.phar -O box.phar
+    wget https://github.com/box-project/box/releases/download/3.9.0/box.phar -O box.phar
```

Comment:
this is giving 404

Reply:
Yeah, I'm not sure why I committed that, anyway we probably won't use 3.9.0 either since it still suffers from the version detection bug. [No need to report the issue though, the team is aware of it](https://github.com/box-project/box/issues/490)